style.css

```diff
@@ -275,17 +275,25 @@ display: none;
   header {
     width: 100%;
     max-width: 90%;
-    
     color: #344563;
+    padding-bottom: 50px;
+    letter-spacing: 0.37px;
+    border-top-right-radius: 120px;
   }
 
   .portfolio-work {
     display: flex;
     flex-direction: row;
     padding: 2% 8%;
     gap: 5%;
-  
-    margin: 24px;
+    margin: 142px 142px 0 142px;
+    align-self: center;
+    padding-left: 25px;
+    width: 80%;
+  }
```

Comment:
- [ ] Kindly complete all the required changes from the last code reviewer to look similar to the Figma template 👇🏽 

> Hey, your images in each card are not of the same size and this is making your design not consistent. Please, ensure all images are of the same size to improve your design. Kindly check the images on Figma to improve on this issue.

>  The second and fourth card content are having a reversed direction from the Figma design. You're yet to add this missing feature. Please, kindly check the card's content closely to see their directions and improve your design.

 

> Kindly maintain the font size, weight, and colors on each card content to improve your design.

>  Kindly maintain the correct styles on the list items in each card to match the Figma design as much as you can.

>  Kindly add the missing interaction states on each button to indicate when it's on hover, active, enabled, disabled. You can check the Figma design to use the right color indicating the different states to improve the user experience of the website.

>  Hey, please, let's adjust our on cards so they have some spaces left and right and not taking the whole width of this project section thereby making it close to the Figma design as much as you can below:

>  Also, kindly remove the space before each image, as they don't seem to have a space in the container. see the images below to improve your design.

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/208250643-1991077b-fb6e-4ccd-bad4-b1e40bcf126a.png) | ![image](https://user-images.githubusercontent.com/58642949/208250623-e94a2e83-e337-4933-bb0a-1a3f45e71d13.png) |